app/src/main/java/org/simple/clinic/drugs/selection/entry/confirmremovedialog/ConfirmRemovePrescriptionDialog.kt

```diff
@@ -45,6 +50,20 @@ class ConfirmRemovePrescriptionDialog : AppCompatDialogFragment(), UiActions {
         .share()
   }
 
+  private val delegate: MobiusDelegate<ConfirmRemovePrescriptionDialogModel, ConfirmRemovePrescriptionDialogEvent, ConfirmRemovePrescriptionDialogEffect> by unsafeLazy {
+    MobiusDelegate.forView(
```

Comment:
Since this is a `Fragment`, we should be using a different saved state handle than the views since state saving and restoration will not work as expected.

We can use the `.forActivity` handle here since their mechanism and fragment's are similar compared to the view's. We should probably rename that to `.forActivityOrFragment` as well, although this can be done later.

I will make this correction since you are on leave.